src/api/Interpret.cc

```diff
@@ -1284,8 +1284,9 @@ void Interpret::getProof()
 
 void Interpret::getUnsatCore() {
     auto unsatCore = main_solver->getUnsatCore();
+    auto & unsatCoreTerms = unsatCore->getTerms();
```

Comment:
```suggestion
    auto const & unsatCoreTerms = unsatCore->getTerms();
```

Reply:
Getterms is already const so auto is const as well. It depends whether getterms could ever be made as non-const as well? But for better readability, I can add the const...